09-redux/classworks/Malkov_Aleksei/src/components/search_bar/SearchBar.js

```diff
@@ -0,0 +1,57 @@
+import React, { useState, useEffect } from 'react';
+import styles from './search-bar.module.css';
+import { ReactComponent as Search } from '../../assets/search.svg';
+import useRedditApi, { withRedditApi } from 'hooks/useRedditApi';
+import { useHistory, useLocation } from 'react-router-dom';
+import { useDispatch } from 'react-redux';
+import getSubreddits from 'components/services/getSubreddits';
+
+
+function useQuery() {
+  return new URLSearchParams(useLocation().search);
+}
+
+function SearchBar({ isSearchPage }) {
+  const [inputSearch, setInputSearch] = useState('');
```

Comment:
it's a bad idea to store the same data in both state and redux. only 1 source of truth